setup.py

```diff
@@ -28,8 +28,8 @@ def make_version_unique(version):
     'entry_points': {
         'console_scripts': ['thriftcli = thriftcli.thrift_cli:main']
     },
-    'install_requires': ['kazoo', 'mock', 'thrift', 'twitter.common.rpc', 'coverage'],
-    'requires': ['kazoo', 'mock', 'thrift', 'twitter.common.rpc', 'coverage'],
+    'install_requires': ['kazoo', 'thrift', 'twitter.common.rpc', 'six'],
```

Comment:
I moved the requirements added for testing purposes to requirements.txt. that way when installing the package for use you dont pull in mock and coverage.